Don't create blank list entry for newline at EOF

JendrBendr opened the pull request:
If a question list is edited using Linux, a newline is appended to the end of the file.  The generate_questions.py script interprets this as a list entry, as we end up with a blank list entry on the flipper.  This fix checks for a newline at the end of the file and removes it if necessary.

JendrBendr committed:
Don't create blank list entry for newline at EOF

If a question list is edited using Linux, a newline is appended to the end of the file.  The generate_questions.py script interprets this as a list entry, as we end up with a blank list entry on the flipper.  This fix checks for a newline at the end of the file and removes it if necessary.

nikilark commented:
Hi, @JendrBendr,
Thank you for your contribution!

How about filtrating empty lines (which might be somewhere in the middle of a file as well), like this

```python
lists[os.path.splitext(os.path.basename(file))[0]] = [line for line in f.read().split('\n') if line and not line.isspace()]
```

What do you think?

JendrBendr commented:
Nice!  That's a way better solution.

nikilark commented:
@JendrBendr, will you do code changes?

JendrBendr committed:
Update generate_questions.py

Much cleaner check by @nikilark that will ignore empty lines both at the end and in the middle of a question list.

JendrBendr commented:
@nikilark Should be all set now.

nikilark commented:
Merged, thank you again, @JendrBendr!
